lib/Listeners/BeforeTemplateRenderedListener.php

```diff
@@ -41,7 +42,7 @@ public function __construct(IUserSession $userSession,
 	public function handle(Event $event): void {
 		if (!($event instanceof BeforeTemplateRenderedEvent)) {
 			return;
-		}
+		};
```

Comment:
This looks a bit weird to me :thinking: 

Leftover/typo?

Reply:
O I misread the diff when adding, thought i had removed that 🙈 Let me drop that

Reply:
Fixed